Add PDF parser template (#115)
airplanedev/templates: pdf_parser/README.md

# Template: PDF parser

## Next steps

- Navigate to the pdf_parser directory: `cd pdf_parser`
- You'll need to create a [OpenAI API key](https://platform.openai.com/account/api-keys) for the PDF parser to work.

> - Once you have your API key from OpenAI, store it the `openai_api_key` [config variable](https://docs.airplane.dev/platform/configs). This will ensure that the task that needs to interact with OpenAI has the proper credentials.

Reviewer: ```suggestion
- Once you have your API key from OpenAI, store it in a [config variable](https://docs.airplane.dev/platform/configs) named `openai_api_key`. This will ensure that the task has the proper credentials it needs to interact with OpenAI.
```

> - Once you have your API key from OpenAI, store it the `openai_api_key` [config variable](https://docs.airplane.dev/platform/configs). This will ensure that the task that needs to interact with OpenAI has the proper credentials.

Reviewer: Can we capitalize the config variable name? I think for most of the other templates, we've standardized on all caps for both the config + env var name, e.g. https://github.com/airplanedev/templates/blob/407cfebd7559e2c1103bf13f1e8e8ae5b87624ae/support_ticket_dashboard/tasks/create_linear_issue.airplane.ts#L49

> - Also add the config variable to your dev config file in order to [develop tasks locally](https://docs.airplane.dev/dev-lifecycle/dev-config-file#add-a-config-variable).

Reviewer: ```suggestion
- Also add the config variable to your [dev config file](https://docs.airplane.dev/dev-lifecycle/dev-config-file#add-a-config-variable) to develop the task locally.
```

> - Uncomment `OPENAI_API_KEY` environment variable in the `pdf_parser.airplane.ts` task.

Reviewer: ```suggestion
- Uncomment the `OPENAI_API_KEY` environment variable in the `pdf_parser.airplane.ts` task.
```

> - Remove the early return statement from the `pdf_parser.airplane.ts` task.

Reviewer: ```suggestion
- Remove the early return statement from the `pdf_parser.airplane.ts` task.
```
This is unnecessary right? Like once `token` is non-empty, we won't hit the return statement anyway.
